wp-content/themes/chuongDemo/sidebar.php

```diff
@@ -0,0 +1,19 @@
+<div class="col-sm-3 col-sm-offset-1 blog-sidebar">
```

Comment:
should get dynamic sidebar. Document is https://developer.wordpress.org/themes/functionality/sidebars/